dotnet/SimpleSauce.Test/BaselineSeleniumAcceptanceTests.cs

```diff
@@ -13,33 +14,47 @@ namespace SauceBindings.Test
     public class BaselineSeleniumAcceptanceTests
     {
         public TestContext TestContext { get; set; }
+        public RemoteWebDriver _driver { get; set; }
 
-        [TestMethod]
-        public void WithoutSauceSession_ShouldWork()
+        [TestInitialize]
+        public void Setup()
         {
             var sauceUserName = Environment.GetEnvironmentVariable("SAUCE_USERNAME");
             var sauceAccessKey = Environment.GetEnvironmentVariable("SAUCE_ACCESS_KEY");
             var sauceOptions = new Dictionary<string, object>
             {
                 ["username"] = sauceUserName,
-                ["accessKey"] = sauceAccessKey
+                ["accessKey"] = sauceAccessKey,
+                ["name"] = nameof(WithoutSauceSession_ShouldWork)
             };
+
             var chromeOptions = new ChromeOptions
             {
                 BrowserVersion = "latest",
                 PlatformName = "Windows 10",
                 UseSpecCompliantProtocol = true
             };
+
             chromeOptions.AddAdditionalOption("sauce:options", sauceOptions);
 
-            var driver = new RemoteWebDriver(new Uri("https://ondemand.saucelabs.com/wd/hub"),
-                chromeOptions.ToCapabilities(), TimeSpan.FromSeconds(30));
-            driver.Navigate().GoToUrl("https://www.google.com");
-            driver.SessionId.Should().NotBeNull();
+            _driver = new RemoteWebDriver(new Uri("https://ondemand.us-west-1.saucelabs.com/wd/hub"),
+                chromeOptions.ToCapabilities(), TimeSpan.FromSeconds(30)); ;
```

Comment:
Codacy found an issue: [Remove this empty statement.](https://app.codacy.com/manual/SauceLabs/sauce_bindings/pullRequest?prid=6319844)